chapter_02_repository.asciidoc

```diff
@@ -538,7 +538,7 @@ TODO: not sure if this diagram is helping.
 ====
 
 
-As always we start with a test. This would probably be classified as an
+As always, we start with a test. This would probably be classified as an
```

Comment:
do you have a rule you're following when adding these commas?

Reply:
I am not too sure, I think the part of the sentence that holds the important bit -what we actually want to say- should be identified by some punctuation (the comma here) that helps us focus on the content, as opposed to the other particules, that are there to pace the sentence maybe?

Reply:
Oh Google helped me there. So the rule is "Use a comma after introductory adverbs" according to this: http://www.businessinsider.com/a-guide-to-proper-comma-use-2013-9

Reply:
It does look better with the comma :+1:

Reply:
the thing is, O'Reilly will have their own style guide and conventions for this sort of thing, so there's a 50-50 chance they're just going to change it back.  When the book goes through the production process a copy-editor goes through the whole thing with a fine-toothed comb, picking out little things like this...